lib/import/rummager_data_presenter.rb

```diff
@@ -0,0 +1,52 @@
+module Import
+  class RummagerDataPresenter
+
+    private_class_method :initialize
+
+    def self.present(row_data)
+
+      rows = []
+
+      base_path     = row_data["link"]
+
+      organisations = fetch_resources(row_data, "organisations", "/government/organisations/")
+      people        = fetch_resources(row_data, "people", "/government/people/")
+      policies      = fetch_resources(row_data, "policies")
+      policy_groups = fetch_resources(row_data, "policy_groups", '/government/groups/')
+      specialist_sectors = fetch_resources(row_data, "specialist_sectors", "/topic/")
+      mainstream_browse_pages = fetch_resources(row_data, "mainstream_browse_pages", "/browse/")
+
+      rows << [base_path, 'policies', encode_array(policies)] if policies
+      rows << [base_path, 'people', encode_array(people)] if people
+      rows << [base_path, 'organisations', encode_array(organisations)] if organisations
+      rows << [base_path, 'working_groups', encode_array(policy_groups)] if policy_groups
+      rows << [base_path, 'topics', encode_array(specialist_sectors)] if specialist_sectors
+      rows << [base_path, 'mainstream_browse_pages', encode_array(mainstream_browse_pages)] if mainstream_browse_pages
+
+      rows
+    end
+
+  private
+
+    def self.encode_array(resource)
+      "{#{resource.join(',')}}"
```

Comment:
we might not need this anymore as this was to mimic postgresql array representation, SQLite doesn't support arrays